docs/debugger/how-to-install-a-visualizer.md

```diff
@@ -28,16 +28,27 @@ After you have created a visualizer, you must install the visualizer so that it
 
 1. Locate the DLL that contains the visualizer you have built.
 
-2. Copy the DLL to either of the following locations:
+2. Copy the [Debugger Side](create-custom-visualizers-of-data.md#to-create-the-debugger-side) DLL to either of the following locations:
 
     - *VisualStudioInstallPath* `\Common7\Packages\Debugger\Visualizers`
 
     - `My Documents\` *VisualStudioVersion* `\Visualizers`
 
-3. If you want to use a managed visualizer for remote debugging, copy the DLL to the same path on the remote computer.
+3. Copy the [Debuggee Side](create-custom-visualizers-of-data.md#to-create-the-debuggee-side) DLL to either of the following locations:
 
-4. Restart the debugging session.
+    - *VisualStudioInstallPath* `\Common7\Packages\Debugger\Visualizers\<FRAMEWORK>`
+
+    - `My Documents\` *VisualStudioVersion* `\Visualizers\<FRAMEWORK>`
+
+    Where `<FRAMEWORK>` is either:
+    - `net2.0` for debuggees running the `.NET Framework` runtime.
+    - `netstandard2.0` for debuggees using a runtime supported in `netstandard 2.0`.
+    - `netcoreapp` for debuggees running the `.NET Core` runtime.
+
```

Comment:
We should probably also say something like the following. Note that I didn't check when Ramkumar added support for the subdirectory, so that version number may be wrong --

If your visualizer is intended to **ONLY** support .NET Framework. The `<FRAMEWORK>` sub-directory may be omitted (for example: install to `My Documents\` *VisualStudioVersion* `\Visualizers`). Versions of Visual Studio before Visual Studio 2019 will only load from this directory.